td|&
__gmon_start__
_init
_fini
_ITM_deregisterTMCloneTable
_ITM_registerTMCloneTable
__cxa_finalize
_Jv_RegisterClasses
_Py_NoneStruct
PyExc_ValueError
PyErr_SetString
PyArg_ParseTuple
PySequence_Check
PySequence_Size
PyList_New
PySequence_GetItem
PyInt_AsLong
PyList_Append
initmultiarray_tests
Py_InitModule4
PyImport_ImportModule
PyObject_GetAttrString
PyCObject_Type
PyExc_RuntimeError
PyExc_ImportError
PyErr_Print
PyCObject_AsVoidPtr
PyErr_Occurred
PyErr_Format
PyExc_AttributeError
libpython2.7.so.1.0
libpthread.so.0
libc.so.6
_edata
__bss_start
_end
GLIBC_2.1.3
d$([
d$(1
UWVS
[^_]
D$@t
D$<t
L$,;O
D$P;F
UWVS
D$<tH
[^_]
D$Pt
D$<1
L$0;O
L$D;H
L$0;O
L$X;H
L$0;O
L$D;H
d$ [^_
d$ [^_
hook/data was not the expected test hook
malloc count is zero after test
bounds sequence 1 size not compatible with x input
bounds sequence 2 size not compatible with x input
bounds sequence size not compatible with x input
numpy.core.multiarray failed to import
_ARRAY_API is not PyCObject object
module compiled against ABI version %x but this version of numpy is %x
module compiled against API version %x but this version of numpy is %x
FATAL: module compiled as unknown endian
FATAL: module compiled as little endian, but detected different endianness at runtime
cannot load umath_tests module.
test_neighborhood_iterator_oob
test_pydatamem_seteventhook_start
test_pydatamem_seteventhook_end
free count is zero after test
OOiOi
bound not long
Type not supported
OOOi
multiarray_tests
numpy.core.multiarray
_ARRAY_API
_ARRAY_API not found
_ARRAY_API is NULL pointer
test_neighborhood_iterator
;*2$"
multiarray_tests.so.debug
7zXZ
k@f\
Bw1<.
3!?=
!3.oI>((
R!"Iwn
.shstrtab
.note.gnu.build-id
.gnu.hash
.dynsym
.dynstr
.gnu.version
.gnu.version_r
.rel.dyn
.rel.plt
.init
.text
.fini
.rodata
.eh_frame_hdr
.eh_frame
.init_array
.fini_array
.jcr
.data.rel.ro
.dynamic
.got
.got.plt
.data
.bss
.gnu_debuglink
.gnu_debugdata
